main/main.c

```diff
@@ -116,6 +122,27 @@ void guiTask(void *pvParameter) {
     ESP_ERROR_CHECK(esp_timer_create(&periodic_timer_args, &periodic_timer));
     ESP_ERROR_CHECK(esp_timer_start_periodic(periodic_timer, LV_TICK_PERIOD_MS * 1000));
 
+    /* Create the demo application */
+    create_demo_application();
+    
+    while (1) {
+        vTaskDelay(LV_TICK_PERIOD_MS);
```

Comment:
Wouldn't be better to use `vTaskDelay(1)` to sleep for one heartbeat?
Now it's the same as `LV_TICK_PERIOD_MS == 1` but looks like someone though that `vTaskDelay` gets `ms` as parameter.

Reply:
Does df5f3385eea49f050e416a77848dc12c15a793ac looks good?

Reply:
I think it will be solved as `vTaskDelay(0)` if tick period is 10 ms. According to [this](https://www.freertos.org/FreeRTOS_Support_Forum_Archive/December_2012/freertos_vTaskDelay_0_vs_vTaskDelay_1_6518023.html) `vTaskDelay(0)` is equivalent to `vTaskYield()`. Is this the desired behaviour?

Reply:
The desired behavior is to delay for 1ms, that's why I used the macro to calculate the rtos ticks, `vTaskDelay` parameter is un ticks. Do you want me to change it to 1?

Reply:
Maybe I miss something but can it delay less than the tick period?

Reply:
I think the delay is for the LVGL tick period not the RTOS tick period.

Reply:
I'm confused now 😄, `vTaskDelay` parameter is in RTOS ticks by using the `pdMS_TO_TICKS` we convert `ms` to RTOS ticks.

Reply:
I see but if RTOS tick is 10 ms (ESP default) `pdMS_TO_TICKS(1)` will be `0`, isn't it?